<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">QTableView::item {
	background: rgb(203, 255, 105);
}

QTableView::item::hover {
	background: teal
}

QTableView {
	selection-color: red;
	font-weight: bold;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonInsert">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(50, 225, 50);
	border: 2px solid rgb(69, 73, 75);
	border-radius: 5px;
	padding-right: 10px;
	padding-left: 10px;
	font-size: 18px;
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Добавить человека</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(255, 75, 75);
	border: 2px solid rgb(69, 73, 75);
	border-radius: 5px;
	padding-right: 10px;
	padding-left: 10px;
	font-size: 18px;
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Удалить человека</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
